zygrader/grader.py

```diff
@@ -394,12 +394,17 @@ def watch_students(student_list, students, lab, use_locks):
                                     lab, use_locks, student_select_fn)
 
 
-def lab_select_fn(selected_index, use_locks):
+def lab_select_fn(selected_index, use_locks, student: model.Student=None):
     """Create the list of labs to pick a student to grade"""
     window = ui.get_window()
     lab = data.get_labs()[selected_index]
     students = data.get_students()
 
+    # A student is already selected, open the grader
+    if student:
+        student_select_fn(student, lab, use_locks)
+        return
+
```

Comment:
I feel like this could be reordered to not allocate variables that don't need to be used if the student is passed in (just putting `window` and `students` after the conditional. for both efficiency (admittedly not a huge concern) and readability, that would help.

the doc string is also strange. it wasn't changed here, but while we're editing in the area, we could probably fix that. this function does not "Create the list of labs", so that threw me off for a bit trying to understand what was going on.

Reply:
Ah good catch. Will do!